src/clusterStatusCheck.ts

```diff
@@ -1,10 +1,10 @@
 import {ssmCommand} from './utils/ssmCommand';
 import {StandardOutputContent} from 'aws-sdk/clients/ssm';
 
-export async function handler(instanceId: string): Promise<string> {
+export async function handler(instanceId: string): Promise<Object> {
```

Comment:
Minor: Could we lean on TS to declare some extra details about the object?

```
interface ClusterStatusResponse {
    instanceId: string;
    clusterStatus: string;
}
```

Then:
```
export async function handler(instanceId: string): Promise<ClusterStatusResponse> {
```

Reply:
👍 for declaring types for our inputs/outputs. I think we're planning to pass the instanceId all the way through, perhaps we can revisit this later

Reply:
Agree with this. Let's make use of types when we can. Should help future explorers.

Reply:
Great idea @katebee! I will raise a separate PR which introduces types for the inputs/outputs of all lambdas, since this will be a project wide improvement.